Tests/PureeTests/Output/BufferedOutputDispatchQueueTests.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import XCTest
+@testable import Puree
+
+private func makeLog() -> LogEntry {
+    return LogEntry(tag: "foo", date: Date())
+}
+
+class BufferedOutputDispatchQueueTests: XCTestCase {
```

Comment:
Why need you add a new test case?

I think it is enough to add test cases to existing `BufferedOutputTests`.

Reply:
The swiftlint configuration has a limit of 400 lines per file.

Adding a new test case exceeded that so I had to move it into a new file for Travis to run the tests

Reply:
I got.

I think this violation should be disabled.